packages/worker/src/transcribe.ts

```diff
@@ -222,12 +222,14 @@ const transcribeAndTranslate = async (
 		const dlParams = whisperParams(true, whisperBaseParams.wavPath);
 		const { metadata } = await runWhisper(whisperBaseParams, dlParams);
 		const languageCode =
-			languageCodes.find((c) => c === metadata.detectedLanguageCode) || 'en';
+			languageCodes.find((c) => c === metadata.detectedLanguageCode) || 'auto';
 		const transcription = await runTranscription(
 			whisperBaseParams,
 			languageCode,
 			false,
 		);
+
```

Comment:
could you add a comment here - something like `we only run language detection once, so need to override the detected language of future whisper runs`